package com.sfsymbols.data.dualtone

import androidx.compose.ui.graphics.vector.ImageVector
import com.sfsymbols.data.SfSymbols
import com.sfsymbols.data.addSfPath
import com.sfsymbols.data.sfIcon

/**
 * SF Symbol: SFArrowDownForwardSquareFill (dualtone)
 * Viewport: 23.3203 x 22.959
 */
public val SfSymbols.Dualtone.SFArrowDownForwardSquareFill: ImageVector
    get() {
        if (_sFArrowDownForwardSquareFill != null) {
            return _sFArrowDownForwardSquareFill!!
        }
        _sFArrowDownForwardSquareFill = sfIcon(
            name = "Dualtone.SFArrowDownForwardSquareFill",
            viewportWidth = 23.3203f,
            viewportHeight = 22.959f
        ) {
            addSfPath("M3.79883 22.959L19.1504 22.959C21.6797 22.959 22.959 21.6797 22.959 19.1992L22.959 3.76953C22.959 1.2793 21.6797 0 19.1504 0L3.79883 0C1.2793 0 0 1.26953 0 3.76953L0 19.1992C0 21.6992 1.2793 22.959 3.79883 22.959Z", fillAlpha = 0.2125f)
            addSfPath("M14.2285 15.2637C15.0684 16.0449 16.0938 15.0781 15.2637 14.209L13.2324 12.0508L7.99805 6.79688C7.83203 6.64062 7.62695 6.54297 7.34375 6.54297C6.85547 6.54297 6.5332 6.86523 6.5332 7.37305C6.5332 7.59766 6.64062 7.8125 6.80664 7.97852L12.0605 13.2227ZM14.8047 11.7969L15.0977 15.1074L12.0215 14.8047L9.20898 14.8047C8.67188 14.8047 8.33008 15.127 8.33008 15.625C8.33008 16.1133 8.66211 16.4355 9.18945 16.4355L15.5273 16.4355C16.084 16.4355 16.4453 16.1621 16.4453 15.5176L16.4453 9.21875C16.4453 8.69141 16.1133 8.33008 15.6348 8.33008C15.1367 8.33008 14.8047 8.66211 14.8047 9.19922Z", fillAlpha = 0.85f)
        }
        return _sFArrowDownForwardSquareFill!!
    }

private var _sFArrowDownForwardSquareFill: ImageVector? = null
